### PR TITLE
Use body element as anchored element for frame lightboxes

### LINKED ISSUE #9972: Confirm need for intermediate container for amp-lightbox in A4A

### PR DESCRIPTION
**Pull request by alanorozco**
Fixes #9972

### CODE REVIEW

File: src/friendly-iframe-embed.js [hunk 1]
```diff
@@ -511,29 +516,22 @@ export class FriendlyIframeEmbed {
...
     return this.runVsyncOnIframe_({
       measure: state => {
         const iframeRect = this.iframe./*OK*/getBoundingClientRect();
 
-        const winWidth = this.win./*OK*/innerWidth;
-        const winHeight = this.win./*OK*/innerHeight;
-
-        state.fixedContainerStyle = {
+        state.bodyStyle = {
+          'background': 'transparent', // TODO(alanorozco): do this early
```
Comment (lannka) [inline src/friendly-iframe-embed.js]: can you explain a bit more in the TODO why "transparent" needs to be early?

Comment (lannka) [inline src/friendly-iframe-embed.js]: do you plan to talk about body transparency in a separate PR (in a detailed README)?

**Comment by alanorozco** [comment]
@lannka Exactly, that's what I meant with the TODO :)

File: src/friendly-iframe-embed.js (hunk 1 above)
Comment (alanorozco) [inline src/friendly-iframe-embed.js]: Because of the warning you mention in your other comment.

Comment (alanorozco) [inline src/friendly-iframe-embed.js]: Ditto.

File: src/style.js [hunk 2]
```diff
@@ -92,6 +92,18 @@ export function getVendorJsPropertyName(style, camelCase, opt_bypassCache) {
 
 
 /**
+ * Sets the CSS styles of the specified element with !important. The styles
+ * are specified as a map from CSS property names to their values.
+ * @param {!Element} element
+ * @param {!Object<string, *>} styles
+ */
+export function setImportantStyles(element, styles) {
+  element.style.cssText +=
```
Comment (jridgewell) [inline src/style.js]: Why isn't this using `setStyles`?

Comment (alanorozco) [inline src/style.js]: Because `!important` will not work using style property access.

Comment (jridgewell) [inline src/style.js]: Ah, ok. Looking into it, you can pass `"important"` as the third argument to `CSSStyleDeclaration#setProperty`, which should avoid a large amount of the parsing this requires. Mind switching to that?

Comment (alanorozco) [inline src/style.js]: Cool, #10824

**Pull request merged**